controllers/index.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const router = express.Router();
+
+const userRoutes = require("./userController.js")
+router.use("/api/users",userRoutes)
```

Comment:
Should be using the "./api" folder and routing `/api`
eg: `router.use("/api", require("./api));`